project-helper.ts

```diff
@@ -55,5 +56,15 @@ export class ProjectHelper implements IProjectHelper {
 		let sanitizedName = _.filter(appName.split(""), (c) => /[a-zA-Z0-9]/.test(c)).join("");
 		return sanitizedName;
 	}
+	
+	private isProjectFileCorrect(projectFilePath: string): boolean {
+		if(this.$staticConfig.CLIENT_NAME_KEY_IN_PROJECT_FILE) {
+			var fileContent = this.$fs.readJson(projectFilePath).wait();
```

Comment:
This code could throw, should we catch it here and return false, or let uplevel  code handle it?